### ISSUE TITLE
Version info in app outdated

### ISSUE THREAD

**Issue by ndeet** [boltcard/bolt-nfc-android-app#49]
The version info on the (i) page is outdated, I thought the app store might have an outdated version so I downloaded 0.3.2 from github but it still shows 0.1.9 on that screen:
![image](https://github.com/user-attachments/assets/1c5e6313-2b08-4a27-a1b6-c4e1fde7564e)

**Comment by robertclarkson** [comment]
Yeah you're right - we need to link that version to the real version data instead of letting it dangle.
